projects/aca-content/src/lib/components/details/details.component.html

```diff
@@ -8,18 +8,21 @@
     <div class="acs-details-container">
       <div class="acs-details-title">
         <div class="acs-details-breadcrumb" role="heading" aria-level="2" *ngIf="node">
-          <span class="acs-details-breadcrumb-library"> {{ node.name }} </span>
-          -
-          <span class="acs-details-breadcrumb-item">{{ 'APP.INFO_DRAWER.TITLE' | translate }}</span>
+          <span class="acs-details-breadcrumb-library">
+            <img class="acs-details-breadcrumb-icon" alt="{{ 'APP.INFO_DRAWER.ICON' | translate }}" src="{{ getNodeIcon(node) }}">       
+            {{ node.name }} </span>
```

Comment:
Use translate for the node name.

Reply:
No please don't do it, `node.name` is a node property not anything that would be a part of our translations so it won't ever be translated..